test/end2end_test.go

```diff
@@ -5535,3 +5535,23 @@ func testServiceConfigMaxMsgSizeTD(t *testing.T, e env) {
 		t.Fatalf("%v.Send(%v) = %v, want _, error code: %s", stream, sreq, err, codes.ResourceExhausted)
 	}
 }
+
+func TestMethodFromServerStream(t *testing.T) {
+	defer leakcheck.Check(t)
+	e := tcpClearRREnv
+	te := newTest(t, e)
+	var method string
+	var ok bool
+	te.unknownHandler = func(srv interface{}, stream grpc.ServerStream) error {
+		method, ok = grpc.MethodFromServerStream(stream)
+		return nil
+	}
+
+	te.startServer(nil)
+	defer te.tearDown()
+	_ = te.clientConn().Invoke(context.Background(), "foo/bar", nil, nil)
+	if !ok || method != "foo/bar" {
+		t.Fatalf("Invoke with method \"foo/bar\", got %s, %v, want \"foo/bar\", true", method, ok)
```

Comment:
Nit: define a testing const, and print `%q` to avoid escaping `"`s.

```go
const testMethod = "/package.service/method"

if !ok || method != testMethod {
￼		t.Fatalf("Invoke with method %q, got %q, %v, want %q, true", testMethod, method, ok, testMethod)
}
```